source/mir/bignum/integer.d

```diff
@@ -364,6 +475,29 @@ struct BigInt(size_t maxSize64)
         return overflow;
     }
 
+    template opBinary(string op)
```

Comment:
No need for BigInt. BigInt should have binary operations because it is fat structure.